client/src/pages/Plan/ItemTypes.js

```diff
@@ -1,3 +1,4 @@
 export const ItemTypes = {
     COURSE_PILL: "coursePill",
+    COURSE_TITLE: "courseTitle",
```

Comment:
Should this be named `COURSE_TILE` instead of title

Reply:
dun